app/views/sandbox/_metric_option.html.erb

```diff
@@ -1,6 +1,5 @@
 <div class="form-group">
-  <%= label_tag do %>
-      <%= check_box_tag metric.to_s, params[metric], params[metric].present? %>
-      <%= metric.to_s.titleize %>
+  <%= label_tag metric.to_s do %>
+      <%= select_tag metric.to_s, options_for_select(@all_metrics, :include_blank => 'name of your blank prompt') %>
```

Comment:
I would extract this into a helper, I would delete the instance variable @all_metrics from the controller, and prepare the data in the helper method.

The include_blank option also need updating.